### DISCUSSION TITLE
Welcome to react-native-swiper-flatlist Discussions!

### DISCUSSION THREAD

**Discussion by gusgard** [gusgard/react-native-swiper-flatlist#91] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kambie10** [comment]
how to make multi items ?

**Reply by gusgard** [reply]
I don't follow you, what do you want to achieve exactly ?

**Comment by MSetnik** [comment]
Would be great to implement FlashList instead of flatlist, the plugin is having performance issues with large image list
